src/App.js

```diff
@@ -1,5 +1,34 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { MESSAGES } from "./constants/message.js";
+import { isPlayerInputValid, isTrialInputValid } from "./utils/validation.js";
+import move from "./utils/move.js";
+import printResult from "./utils/printResult.js";
+
 class App {
-  async play() {}
+  async play() {
+    // 게임 시작
+    const playerInput = await MissionUtils.Console.readLineAsync(MESSAGES.START_INPUT);
+    const players = isPlayerInputValid(playerInput);
+    const playersMap = new Map(players.map((player) => [player, 0]));
+
+    const tryInput = await MissionUtils.Console.readLineAsync(MESSAGES.TRY_INPUT);
+    const tryNum = isTrialInputValid(tryInput);
```

Comment:
```suggestion
    const tryNum = isTrialInputValid(tryInput);
```
`isTrialInputValid` 함수도 `boolean` 값을 반환하지 않는데 `is`라는 접두사를 사용해서 어색한 느낌이 있는 것 같아요🤔

Reply:
하나의 함수에서 여러가지를 할 수 있도록 하고 싶었는데 욕심이었던 것 같네요! 함수의 기능은 명확할수록 좋은건데..ㅠ 리뷰 감사합니당